src/cmdlib.sh

```diff
@@ -186,6 +186,10 @@ prepare_build() {
     manifest=${configdir}/manifest.yaml
     manifest_lock=${configdir}/manifest-lock.${basearch}.json
     manifest_lock_overrides=${configdir}/manifest-lock.overrides.${basearch}.json
+    if [ -f "${manifest_lock_overrides/json/yaml}" ]; then
+        manifest_lock_overrides=${manifest_lock_overrides/json/yaml}
+    fi
+
```

Comment:
https://github.com/coreos/fedora-coreos-config/pull/227#issuecomment-552671968
can we add support for yaml for the `$manifest_lock` as well?

Reply:
another thing to consider here is that if both a json and yaml exist then the yaml will be used and the json will be ignored. One could argue that if they both exist then we should error out.

Reply:
Agree on erroring out if we find both!